src/classes/PS_IntegrationService.cls

```diff
@@ -35,16 +35,113 @@
 * @description Provides a public interface to the Payment Services Integration Package
 */
 global class PS_IntegrationService {
-    
+
+    @TestVisible private static final String PAYMENTS_APPLICATION_TYPE = 'payments';
```

Comment:
Why do we have the payments and makana configurations? The api key and the base url are duplicated. Can we just use the payments configuration?